src/oauth-helpers.js

```diff
@@ -7,7 +7,9 @@ import { ORBIT_HEADERS } from "./constants";
  *
  * @returns {Object} API_TOKEN WORKSPACE ACCESS_TOKEN REFRESH_TOKEN EXPIRES_AT
  */
-export async function getOrbitCredentials() {
+export async function getOrbitCredentials(
```

Comment:
Making a mental note to go back to this next week (once the user-facing stuff has been tackled).

I think I understand why you needed that callback here: sometimes this method is called from inside a tab, and sometimes from the background script, and the bahavior is different with both. It makes reading the code a bit complex though, because as a caller I know have to know _how the method works_ to call it.

Ideally, the code would look like this:

```javascript
// in background.js, and options.js
// I don’t need to know how the sausage is made, token refresh is not my concern as a caller
const ORBIT_CREDENTIALS = await getOrbitCredentials();
```

No rush though, we can pair on this next week! It might become easier once the old code has been removed too, since we’ll only have `options.js` and `background.js` to deal with.